test/behavioral/tests/tabs.ts

```diff
@@ -1,13 +1,15 @@
-import {Snappit, WebDriver, Key} from "snappit-visual-regression";
+import {WebDriver, By, Key} from "snappit-visual-regression";
 import {test, TestContext} from "ava";
 import * as util from "../../common/util";
 
 const getTab = async (
     driver: WebDriver,
-    tabName: string
+    tabID: string
 ) => {
-    const tab = await util.$x(driver, util.selectors.tab, tabName);
-    return tab;
+    await driver.findElement(By.css("hx-tablist")).click();
+    await driver.findElements(By.css("hx-tab"));
+    const element = await driver.findElement(By.id(tabID));
+    return element;
 }
 
 const cycleLeft = async (driver: WebDriver) => {
```

Comment:
_(outside current scope of work)_

The `cycleLeft()` and `cycleRight()` functions would be more useful as a generic behavioral utilities (applicable to tests other than tabs).